tools/bwa.wdl

```diff
@@ -34,8 +34,8 @@ task bwa_aln {
         File bwa_db_tar_gz
         String prefix = sub(
             basename(fastq),
-            "([_\\.][rR][12])?(\\.subsampled)?\\.(fastq|fq)(\\.gz)?$",
-            ""
+            "(([_.][rR](?:ead)?[12])((?:[_.-][^_.-]*?)*?))?\\.(fastq|fq)(\\.gz)?$",
```

Comment:
If we're going to go with something this complex, I think it needs to be stated in plain English somewhere. Not necessarily copied to every file, but somewhere accessible. Maybe each usage has a comment like "See `../../regex.md`" or something.